qt/qssl_wolf/tst_wolfssl.cpp

```diff
@@ -0,0 +1,3676 @@
+/* QSslSocket program for wolfSSL verification */
```

Comment:
Did we write this file?  If so, we should add our standard wolfSSL copyright in the header comment.

Reply:
The code is bases on qt unit test code, which is qsslsockt. Many parts of the code are copied from it, and then modified to not use qt test server for convenience. In this case, can we still place our standard copyright to the file?

Reply:
Ideally, we would have both Qt's copyright header and the wolfSSL header if the file contains code owned by both parties.  Since this is test code, and was mainly taken from qsslsocket, leaving our header off is ok.